front/src/modules/simulationResult/components/TrainDetails.tsx

```diff
@@ -30,33 +31,48 @@ const getOccupancyBounds = (
 };
 
 export default function TrainDetails() {
-  const { positionValues, timePosition, consolidatedSimulation, selectedTrainId } = useSelector(
+  const { consolidatedSimulation, selectedTrainId } = useSelector(
     (state: RootState) => state.osrdsimulation
   );
 
+  const [localTimePosition, setLocalTimePosition] = useState<Date>(new Date());
+  const [localPositionValues, setLocalPositionValues] = useState<PositionsSpeedTimes<Date>>(
```

Comment:
You can replace localPositionValue by headPosition, tailPosition, speed, I think it makes it easier to read in the component
```suggestion
  const [{ headPosition, tailPosition, speed }, setLocalPositionValues] = useState<PositionsSpeedTimes<Date>>(
```

Reply:
done